### PR TITLE
added a legend to the leaderboard (resolving #9)

### PR DESCRIPTION
**Pull request by LeMoonStar**
image

### CODE REVIEW

**Comment by LeMoonStar** [comment]
well... I selected to push to unstable, but it selected master automatically again........ why does GitHub do this so often? and where can I change it?

File: src/Leaderboard.ts [hunk 1]
```diff
@@ -149,6 +149,9 @@ export default class Leaderboard {
         }`
       )
       .setDescription(`Next update: ${nextUpdate.toLocaleTimeString()}`)
+      .setDescription(
+        `:new_moon:: No part of the day is completed\n:last_quarter_moon:: Part 1 out of 2 is completed\n:star2:: Both, part 1 and part 2 were completed during the last hour\n:star:: Both part 1 and part 2 are completed\n:sparkles:: Both part 1 and part 2 were completed within the first 3 hours the challange was online.\n\nNext update: ${nextUpdate.toLocaleTimeString()}`
```
Comment (TrojanerHD) [inline src/Leaderboard.ts:153]: 1.
```suggestion
        `:new_moon:: No part of the day is completed\n:last_quarter_moon:: Part 1 out of 2 is completed\n:star2:: Both part 1 and part 2 were completed during the last hour\n:star:: Both part 1 and part 2 are completed\n:sparkles:: Both part 1 and part 2 were completed within the first 3 hours the challenge was online.\n\nNext update: ${nextUpdate.toLocaleTimeString()}`
```
2. This is a bit confusing: Does that mean you have to solve both part 1 and 2 within an hour to get the star?

File: src/Leaderboard.ts [hunk 2]
```diff
@@ -149,6 +149,9 @@ export default class Leaderboard {
         }`
       )
       .setDescription(`Next update: ${nextUpdate.toLocaleTimeString()}`)
```
Comment (TrojanerHD) [inline src/Leaderboard.ts:151]:
```suggestion

```

**Review by TrojanerHD — CHANGES_REQUESTED** [review]

File: src/Leaderboard.ts (hunk 1 above)
Comment (LeMoonStar) [reply src/Leaderboard.ts:153]: It says "was completed during the __last__ hour". How is it possible that literally everyone thinks this is confusing? it is simple clear english again. __within the last hour__

Comment (LeMoonStar) [reply src/Leaderboard.ts:153]: or should I simply write "signals a new star, temporary and will be replaced one hour after completion of the day"?

File: src/Leaderboard.ts (hunk 2 above)
Comment (LeMoonStar) [reply src/Leaderboard.ts:151]: I honestly don't know why that is there, I could swear I removed it...

**Review by TrojanerHD — APPROVED** [review]

**Pull request merged**